### PR TITLE
Add T1539 macOS Chrome Remote Debugging

### PR DESCRIPTION
**Pull request by jonod8698**
**Details:**
New macOS test for using Chrome Remote Debugging mode to dump Chrome cookies. Chrome is started in remote debugging mode and [WhiteChocolateMacademiaNut by Justin Bui from SpectreOps](https://posts.specterops.io/hands-in-the-cookie-jar-dumping-cookies-with-chromiums-remote-debugger-port-34c4f468844e) is used to connect to the remote debugging port and dump user cookies.

**Testing:**
Installing Prerequisites
image

Executing the test #1
image

Executing the test #2 (consecutively)
image

Cleanup
image

**Associated Issues:**
N/A

### CODE REVIEW

File: atomics/T1539/T1539.yaml [hunk 1]
```diff
@@ -77,3 +77,38 @@ atomic_tests:
...
+  - macos 
+  dependency_executor_name: bash
+  dependencies:
+  - description: 'go and whitechocolatemacadamianut '
+    prereq_command: |-
+      go version
+      /tmp/WhiteChocolateMacademiaNut/chocolate -h
+    get_prereq_command: |-
+      brew install go --force
+      git clone https://github.com/slyd0g/WhiteChocolateMacademiaNut.git
```
Comment (cyberbuff) [inline atomics/T1539/T1539.yaml:100]: Can you either add `cd /tmp` or clone into `/tmp/WhiteChocolateMacademiaNut` ?

File: atomics/T1539/T1539.yaml [hunk 2]
```diff
@@ -77,3 +77,38 @@ atomic_tests:
...
+  supported_platforms:
+  - macos 
+  dependency_executor_name: bash
+  dependencies:
+  - description: 'go and whitechocolatemacadamianut '
+    prereq_command: |-
+      go version
+      /tmp/WhiteChocolateMacademiaNut/chocolate -h
+    get_prereq_command: |-
+      brew install go --force
```
Comment (cyberbuff) [inline atomics/T1539/T1539.yaml:99]: Can you also split the requirements into two dependencies ? One to install `go` and another one to clone `WhiteChocolateMacademiaNut` ?

**Review by cyberbuff — CHANGES_REQUESTED** [review]
Thanks for your first time contribution. Welcome to the project. Everything looks good. Just some minor corrections

File: atomics/T1539/T1539.yaml (hunk 1 above)
Comment (jonod8698) [reply atomics/T1539/T1539.yaml]: Done!

File: atomics/T1539/T1539.yaml (hunk 2 above)
Comment (jonod8698) [reply atomics/T1539/T1539.yaml]: Split the two out into separate dependencies.

**Review by cyberbuff — APPROVED** [review]
Works great. Thanks for your contribution. Keep it up 👍

**Pull request merged**